core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1502,6 +1505,30 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets whether the response header will include {@code "Server"} header.
+     */
+    public ServerBuilder disableServerHeader() {
+        this.useServerHeader = false;
+        return this;
+    }
+
+    boolean useServerHeader() {
```

Comment:
@trustin Is this our normal style for boolean getter? It really confuses me since it looks like it should actually mutate the field to `true`

Reply:
I tried to use same style of this class.
Like this,
https://github.com/line/armeria/blob/59d98c355b22aad1a566e94aefafd3848ae20dee/core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java#L1430
But, I agree that it's confusing.
I think `useServerHeader()` sounds like `setter` because it start with `use`.
What about `IsServerHeaderEnabled()`? or any good name?

Reply:
`isServerHeaderEnabled` sounds better to me.

Reply:
Renamed to `isServerHeaderEnabled` and `isDateHeaderEnabled`.